sdk/ml/azure-ai-ml/azure/ai/ml/automl/_automl_nlp.py

```diff
@@ -46,6 +46,15 @@ def text_classification(
 
     :return: The TextClassificationJob object.
     :rtype: TextClassificationJob
```

Comment:
@bhimar Could you please add the full paths whenever we have an azure-ai-ml class in the docstring? (e.g. `azure.ai.ml.automl.TextClassificationJob` instead of `TextClassificationJob`)